website/src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,208 @@
+import React, {useState, useRef, useCallback, useMemo} from 'react';
```

Comment:
Note that this file was ["swizzled"](https://docusaurus.io/docs/swizzling) from a Docusaurus core class, because there doesn't seem to be any other way to create context-aware search rankings.  For the actual change we made to the docusaurus core class, see https://github.com/cursorless-dev/cursorless/pull/915/commits/5a2a365ad35de713adec4661b3bc1c291c3db4a9